src/main/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImpl.java

```diff
@@ -186,16 +192,31 @@ public boolean routeAdvanceDepositDocuments() {
                     LOG.info("Routing advance deposit document # " + documentId + ".");
                 }
                 documentService.routeDocument(advanceDocument, "document routed by achIncome batch job", null);
-            } catch (WorkflowException e) {
-                LOG.error("Error routing document # " + documentId + " " + e.getMessage());
-                throw new RuntimeException(e.getMessage(), e);
+            }
+            catch (Exception e) {
+                LOG.error("Error routing document # " + documentId + " due to exception: " + e.getMessage());
+                logException(e);
             }
         }
 
         return true;
+    }  
+
+	private void logException(Exception e) {
+        if (e instanceof ValidationException) {
```

Comment:
not quite sure why it looks off, it looks fine in Eclipse. Maybe because it is using spaces rather then tabs...